doc/api/http2.md

```diff
@@ -3003,23 +3003,35 @@ The `name` property of the `PerformanceEntry` will be equal to either
 If `name` is equal to `Http2Stream`, the `PerformanceEntry` will contain the
 following additional properties:
 
+* `id` {number} The identifier of the associated `Http2Stream`
+* `receivedBytes` {number} The number of DATA frame bytes received for this
+  `Http2Stream`.
+* `sentBytes` {number} The number of DATA frame bytes sent for this
+  `Http2Stream`.
 * `timeToFirstByte` {number} The number of milliseconds elapsed between the
   `PerformanceEntry` `startTime` and the reception of the first `DATA` frame.
+* `timeToFirstByteSent` {number} The number of milliseconds elapsed between
+  the `PerformanceEntry` `startTime` and sending of the first `DATA` frame.
 * `timeToFirstHeader` {number} The number of milliseconds elapsed between the
   `PerformanceEntry` `startTime` and the reception of the first header.
 
 If `name` is equal to `Http2Session`, the `PerformanceEntry` will contain the
 following additional properties:
 
+* `framesReceived` {number} The number of HTTP/2 frames received by the
+  `Http2Session`.
+* `framesSent` {number} The number of HTTP/2 frames sent by the `Http2Session`.
+* `maxConcurrentStreams` {number} The maximum number of streams concurrently
+  open during the lifetime of the `Http2Session`.
 * `pingRTT` {number} The number of milliseconds elapsed since the transmission
   of a `PING` frame and the reception of its acknowledgement. Only present if
   a `PING` frame has been sent on the `Http2Session`.
-* `streamCount` {number} The number of `Http2Stream` instances processed by
-  the `Http2Session`.
+* `receivedBytes` {number} The number of bytes received for this `Http2Session`.
+* `sentBytes` {number} The number of bytes sent for this `Http2Session`.
```

Comment:
The `net` & `fs` APIs use `bytesWritten` and `bytesRead` for these values… might be worth being consistent with that?